workshops/shiny_r_adv.qmd

```diff
@@ -1,35 +1,35 @@
 ---
-title: Advanced to Shiny for R
+title: Building Brilliant Shiny Apps in R
```

Comment:
@mine-cetinkaya-rundel I started to wonder if the word _advanced_ in "Advanced Shiny for R" might lead to some people we'd want to reach self-selecting out of the workshop. Here's another take at the title.

That said, I can also see that we have plenty of prior art and maybe even some "brand" recognition for _Advanced Shiny_, so I'd be just as fine sticking with that title.

```suggestion
title: Advanced Shiny for R
```

Reply:
I don't think we have to use "advanced", there used to be a curriculum known by that name but we haven't taught it in some time. I think something that describes a bit  -- that one is called "Getting Started with Shiny for R". "Building" sounds like "from scratch", like you'd be first starting off. This is more of an "improving" workshop. I'll keep thinking too but maybe this helps you come up with a slightly different title?

Reply:
Maybe we could just drop "Building"?

* Brilliant Shiny Apps in R

Here are a few other titles I've brainstormed

* Beyond the Basics with Shiny in R
* Scalable Shiny in R
* Scaling Shiny Apps in R
* Level Up with Shiny in R
* Mastering Shiny in R

Do any of these sound interesting/compelling/like a good fit?

Reply:
I like level up and mastering, and happy with either! Feel free to pick the one you prefer.

Reply:
After some internal discussion with the team, we've decided we like _Level Up with Shiny for R_. I'll update the PR 😄